packages/algorand-js/test/helpers/integration.ts

```diff
@@ -0,0 +1,39 @@
+import * as fs from "fs";
+import path from "path";
+
+/**
+ * This helper adds mocha hooks to run the tests inside one of the projects
+ * from test/integration-test-files.
+ *
+ * @param filename The base name of the folder with the integration files to use.
+ */
+export function useIntegrationFile (filename: string): void {
+  let filePath: string;
+  let prevWorkingDir: string;
```

Comment:
```suggestion
 * This helper adds mocha hooks to run the tests inside one of the projects in test/fixtures.
 * @param fixtureName: The base name of the folder in test/fixtures
 */
export function useFixture (fixtureName: string): void {
  let dirPath: string;
  let prevWorkingDir: string;
```

Reply:
Note: I change the argument from filename to dirname, you need to update the code below as well.

Let's rename the directory as well: 
test/integration-test-files -> test/fixtures

Reply:
ok